react/features/base/config/reducer.js

```diff
@@ -232,6 +232,12 @@ function _translateLegacyConfig(oldValue: Object) {
         };
     }
 
+    if (!oldValue.disableModeratorIndicator
```

Comment:
Since this is a boolean option, check if it's undefined, rather than if it's falsey.